app/jobs/debt/tuition_debt_job.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Debt
+  class TuitionDebtJob < ApplicationJob
+    queue_as :high
+
+    def perform(unit_ids, term_id, due_date)
+      units = Unit.where(id: unit_ids.reject(&:empty?)).select { |u| u.students.active.exists? }
```

Comment:
Burada `reject`'e gerek yok. Zaten where yaptığın için  ve `'', nil` gibi id'ler hiç bir zaman olmayacağından ekstra bir kontrolle gerek yok.